static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -83,7 +84,67 @@ const canShowPreChecks = ({
 let messageConfig: InAppMessage;
 let appboy: ?AppBoy;
 
+const getForcedBrazeMessage = (): InAppMessage | null => {
```

Comment:
We could possibly name this to be more explicit about it coming from the URL parameters, e.g. getMessageFromUrlParams - though not essential!

Reply:
Yeah I think that's a good shout, never hurts to be more explicit.

Reply:
I ended up going with `getMessageFromQueryString`, hope that sounds alright?